### PR TITLE
Logging 5.7.11 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.14, 4.13, 4.12

Issue: https://issues.redhat.com/browse/OBSDOCS-777

Link to docs preview:

QE review: @anpingli / @QiaolingTang / @kabirbhartiRH 
- [x] QE has approved this change.

Additional information:
SME Review: @jcantrill / @xperimental / @alanconway / @periklis

### CODE REVIEW

**Review by periklis — APPROVED** [review]
LGTM

**Comment by kabirbhartiRH** [comment]
@libander Could you please add CVE-2023-39326 under the CVE list. It got added on the errata after ProdSec approved the errata.

File: modules/logging-release-notes-5-7-11.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// module included in logging/logging-5-7-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-7-11_{context}"]
+= Logging 5.7.11
+This release includes link:https://access.redhat.com/errata/RHSA-2024:0694[Logging Bug Fix 5.7.11]
+
+[id="logging-release-notes-5-7-11-bug-fixes"]
+== Bug fixes
+* Before this update, when configured to read a custom S3 Certificate Authority the Loki Operator would not automatically update the configuration when the name of the ConfigMap or the contents changed. With this update, the Loki Operator is watching for changes to the ConfigMap and automatically updates the generated configuration. (link:https://issues.redhat.com/browse/LOG-4968[LOG-4968])
```
Comment (adellape) [inline modules/logging-release-notes-5-7-11.adoc:9]: Comma, markup, and a suggested tweak:
```suggestion
* Before this update, when configured to read a custom S3 Certificate Authority, the Loki Operator would not automatically update the configuration when the name of the `ConfigMap` object or the contents changed. With this update, the Loki Operator now watches for changes to the `ConfigMap` object and automatically updates the generated configuration. (link:https://issues.redhat.com/browse/LOG-4968[LOG-4968])
```

File: modules/logging-release-notes-5-7-11.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// module included in logging/logging-5-7-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-7-11_{context}"]
+= Logging 5.7.11
+This release includes link:https://access.redhat.com/errata/RHSA-2024:0694[Logging Bug Fix 5.7.11]
```
Comment (adellape) [inline modules/logging-release-notes-5-7-11.adoc:5]: Punctuation:
```suggestion
This release includes link:https://access.redhat.com/errata/RHSA-2024:0694[Logging Bug Fix 5.7.11].
```
Also I notice the other entries say more fully "OpenShift Logging", so maybe:
```suggestion
This release includes link:https://access.redhat.com/errata/RHSA-2024:0694[OpenShift Logging Bug Fix 5.7.11].
```

**Comment by adellape** [comment]
> Version(s): 4.14, 4.13, 4.12

Should these be going to 4.15 too for future-proofing, or not yet? Looks like the 4.15 preview is currently a few z-streams behind: https://docs.openshift.com/container-platform/4.15/logging/logging_release_notes/logging-5-7-release-notes.html

File: modules/logging-release-notes-5-7-11.adoc (hunk 2 above)
Comment (libander) [reply modules/logging-release-notes-5-7-11.adoc]: Logging is a legal approved short name, OpenShift Logging is not the current legal name.

**Comment by kalexand-rh** [comment]
@libander, please answer Alex's question about 4.15.

**Comment by libander** [comment]
> > Version(s): 4.14, 4.13, 4.12
> 
> Should these be going to 4.15 too for future-proofing, or not yet? Looks like the 4.15 preview is currently a few z-streams behind: https://docs.openshift.com/container-platform/4.15/logging/logging_release_notes/logging-5-7-release-notes.html

Version(s): 4.14, 4.13, 4.12 is correct.

**Pull request merged**